crates/invoker-impl/src/invocation_task/service_protocol_runner.rs

```diff
@@ -228,10 +229,22 @@ where
         ]);
 
         // Inject OpenTelemetry context
-        TraceContextPropagator::new().inject_context(
-            &Span::current().context(),
```

Comment:
Can't you just attach parent_span_context here with `attach_to_span`?

Reply:
I mean just create the span here

Reply:
we don't want to create a new span under the parent span context and pass that, we want the headers to say that it is in the parent span context, no?

Reply:
I think it depends on the way you see it. One might argue that we want to see the "invocation attempt" span here, hence this span to create.